examples/observability/adot-amp-grafana-for-haproxy/main.tf

```diff
@@ -98,6 +97,32 @@ module "eks_blueprints_kubernetes_addons" {
 # Observability Resources
 #---------------------------------------------------------------
 
+module "managed_grafana" {
```

Comment:
Nice! I will be using this module for EMR on EKS as well to remove the manual steps 👍🏼